R/Report_KRI.R

```diff
@@ -39,14 +42,22 @@ Report_KRI <- function(
   dfResults = NULL,
   dfGroups = NULL,
   dfMetrics = NULL,
-  strOutpath = NULL
+  strOutputDir = fs::path_abs("."), ## or just use getwd() to remove a dependency...
```

Comment:
We need the package dependency anyway, but I think `getwd()` is probably clearer for most users, so I'd go with `getwd()`.